src/main/java/com/avioconsulting/mule/connector/akv/provider/internal/error/AzureKeyVaultErrorType.java

```diff
@@ -7,5 +7,5 @@
   SECRET_NOT_FOUND,
   KEY_NOT_FOUND,
   CERTIFICATE_NOT_FOUND,
-  UNKNOWN_ERROR
```

Comment:
Why are we removing the UNKNOWN_ERROR?

Reply:
This was due to Mulesoft's best practices here: https://docs.mulesoft.com/mule-sdk/1.1/error-handling